src/tests/api/types.ts

```diff
@@ -68,6 +68,7 @@ export interface UserInfoResponse {
 }
 
 export interface SessionConfigResponse {
+	pcl_enabled: boolean;
```

Comment:
can we have this field optional please. We wont set it if the SSM param is missing or not found.